### PR TITLE
Add new django service

### PR DESCRIPTION
**Pull request by winpat**

### CODE REVIEW

**Comment by winpat** [comment]
@hairmare The README seems to be out of date. Do we want to keep maintaining the variable documentation?

File: tasks/python.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  template:
+    src: camac-api.service.j2
+    dest: /etc/systemd/system/camac-api.service
+
+- name: render uwsgi ini
+  template:
+    src: uwsgi.ini.j2
+    dest: "{{ camac_python_confdir }}/uwsgi.ini"
+
+- name: install camac django app dependencies
```
Comment (winpat) [inline tasks/python.yml]: duplicate.

File: tests/test.retry [hunk 2]
```diff
@@ -0,0 +1 @@
+localhost
```
Comment (winpat) [inline tests/test.retry]: this should not be checked in.

**Comment by winpat** [comment]
I've discussed with @czosel that we are going to fix the documentation in a separate MR request.

**Review by czosel — APPROVED** [review]

File: defaults/main.yml [hunk 3]
```diff
@@ -68,23 +69,25 @@ camac_base_domain: "camac.ch"
 # camac base directories #
 ##########################
 
+camac_python_basedir: "/usr/share/camac"
```
Comment (sliverc) [inline defaults/main.yml]: is it a good idea to have the same default for python and php base dir?

File: templates/env.j2 [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+DJANGO_DATABASE_PASSWORD="{{ camac_db_password }}"
+DJANGO_DATABASE_HOST="{{ camac_db_host }}"
+DJANGO_DATABASE_PORT="{{ camac_db_port }}"
+DJANGO_ALLOWED_HOSTS="{{ camac_python_django_allowed_hosts }}"
+DJANGO_SECRET_KEY="{{ camac_python_django_secret_key }}"
+DJANGO_MEDIA_ROOT="{{ camac_python_django_media_root }}"
+DJANGO_DEFAULT_FROM_EMAIL="{{ camac_python_django_default_from_email }}"
+DJANGO_SERVER_EMAIL="{{ camac_python_django_server_email }}"
+DJANGO_EMAIL_HOST="{{ camac_python_django_email_host }}"
+APPLICATION_ENV="{{ camac_env }}"
```
Comment (sliverc) [inline templates/env.j2]: this is only needed for dev purposes and should not be overwritten on a deployment. Better remove it.

**Review by sliverc — CHANGES_REQUESTED** [review]

File: defaults/main.yml (hunk 3 above)
Comment (winpat) [reply defaults/main.yml]: I will check if this variable is actually required.

File: templates/env.j2 (hunk 4 above)
Comment (winpat) [reply templates/env.j2]: This leads to `django.core.exceptions.ImproperlyConfigured: Set the APPLICATION_ENV environment variable`. I will fave a look at the settings.py.

Comment (winpat) [reply templates/env.j2]: `ENV = env.str("APPLICATION_ENV")` in the `settings.py` causes the exception.

Comment (czosel) [reply templates/env.j2]: Probably we should just use
`ENV = env.str("APPLICATION_ENV", "production")`

File: defaults/main.yml (hunk 3 above)
Comment (winpat) [reply defaults/main.yml]: This should be fixed now. I've changed the basedir path of the python service to `/usr/share/camac-api/`.

File: templates/env.j2 (hunk 4 above)
Comment (sliverc) [reply templates/env.j2]: totally agree with @czosel

Comment (winpat) [reply templates/env.j2]: Fixed it.

**Pull request merged**